### ISSUE TITLE
wrong integral with integrate(1/(2 + sin(x)), (x, 0, 2*pi))

### ISSUE THREAD

**Issue by Piruzzolo** [sympy/sympy#11165]
It returns 0, while the correct result is 2*Pi/Sqrt(x) (taken from a textbook on residual's method and checked with Mathematica 10)

**Comment by Piruzzolo** [comment]
It doesn't work without a and b, either.

**Comment by jksuom** [comment]
It is a known issue that definite integrals of periodic functions (whose antiderivatives are multivalued) are not correctly evaluated in general. But it is unexpected that not even the antiderivative is computed unlike that of the variant with `cos` instead of `sin`.

**Comment by oscarbenjamin** [comment]
This seems to work fine now:
```julia
In [20]: integrate(1/(2 + sin(x)), (x, 0, 2*pi))                                                                                                              
Out[20]: 
2⋅√3⋅π
──────
  3 
```

**Comment by jksuom** [comment]
This could have been closed in #13808.
